playground/mocks/data/idp/idx/authenticator-authenticate-remediation-form.json

```diff
@@ -0,0 +1,251 @@
+                  "name": "methodType",
+                  "required": false,
+                  "value": "push",
+                  "mutable": false
+                }
+              ]
+            }
+          },
+          "relatesTo": {
+            "profile": {
+              "deviceName": "sdk_gphone_x86"
+            },
+            "type": "app",
+            "key": "custom_app",
+            "id": "pfd99kq1qDar0nBIj0g4",
+            "displayName": "test Custom Push Authenticator",
+            "methods": [
+              {
+                "type": "push"
+              }
+            ]
+          }
+        },
+        {
+          "label": "test Custom Push Authenticator",
+          "value": {
+            "form": {
+              "value": [
+                {
+                  "name": "id",
+                  "required": true,
+                  "value": "aut7116ORq51MqmMi0g4",
+                  "mutable": false
+                },
+                {
+                  "name": "methodType",
+                  "required": false,
+                  "value": "push",
+                  "mutable": false
+                }
+              ]
+            }
+          },
+          "relatesTo": {
+            "profile": {
+              "deviceName": "sdk_gphone_x86"
+            },
+            "type": "app",
+            "key": "custom_app",
+            "id": "pfd99kxTsEQHi20E70g4",
+            "displayName": "test Custom Push Authenticator",
+            "methods": [
+              {
+                "type": "push"
+              }
+            ]
+          }
+        },
+        {
+          "label": "Security Key or Biometric",
+          "value": {
+            "form": {
+              "value": [
+                {
+                  "name": "id",
+                  "required": true,
+                  "value": "aut9d78ScSLylrQck0g4",
+                  "mutable": false
+                },
+                {
+                  "name": "methodType",
+                  "required": false,
+                  "value": "webauthn",
+                  "mutable": false
+                }
+              ]
+            }
+          },
+          "relatesTo": {
+            "type": "security_key",
+            "key": "webauthn",
+            "id": "aut9d78ScSLylrQck0g4",
+            "displayName": "Security Key or Biometric",
+            "methods": [
+              {
+                "type": "webauthn"
+              }
+            ],
+            "allowedFor": "sso"
+          }
+        },
+        {
+          "label": "Okta Verify",
+          "value": {
+            "form": {
+              "value": [
+                {
+                  "name": "id",
+                  "required": true,
+                  "value": "aut70utI9vVyYhoof0g4",
+                  "mutable": false
+                },
+                {
+                  "name": "methodType",
+                  "type": "string",
+                  "required": false,
```

Comment:
same question here

Reply:
yeah, thats correct. We are changing this in UI to be sent in challenge request to backend.